http/src/request/guild/member/search_guild_members.rs

```diff
@@ -89,7 +85,7 @@ struct SearchGuildMembersFields {
 /// let client = Client::new("my token");
 ///
 /// let guild_id = GuildId(100);
-/// let members = client.search_guild_members(guild_id, String::from("Wumpus")).limit(10)?.await?;
+/// let members = client.search_guild_members(guild_id, String::from("Wumpus")).limit(10)?.exec().await?;
```

Comment:
I wish there was a formatter for examples. I have a feeling this should be wrapped but it's not a huge deal